tests/batch/test_get_batch_status.py

```diff
@@ -0,0 +1,182 @@
+"""
+Provide tests for command line interface's get batch status command.
+"""
+import json
+
+import pytest
+from click.testing import CliRunner
+
+from cli.constants import (
+    FAILED_EXIT_FROM_COMMAND_CODE,
+    NODE_IP_ADDRESS_FOR_TESTING,
+    PASSED_EXIT_FROM_COMMAND_CODE,
+)
+from cli.entrypoint import cli
+from cli.utils import dict_to_pretty_json
+
+
+def test_get_batch_status():
+    """
+    Case: get a batch status by its identifier.
+    Expect: batch status is returned.
+    """
+    batch_id = '6f200995e766da7218ec2a3d0aeabbe1151128063cdf4e954cd08390a879b28e' \
+               '085a06f8708d2e6bb34f6501e8ddc981f0353627c1d4f90c80a656a8090c8751'
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'batch',
+        'get-status',
+        '--id',
+        batch_id,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert isinstance(json.loads(result.output), dict)
+
+
+def test_get_batch_status_with_invalid_id():
+    """
+    Case: get a batch status by invalid identifier.
+    Expect: the following identifier is invalid error message.
+    """
+    invalid_batch_id = 'abcefg'
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'batch',
+        'get-status',
+        '--id',
+        invalid_batch_id,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    expected_error_message = {
+        'errors': {
+            'id': [
+                f'The following identifier `{invalid_batch_id}` is invalid.',
+            ],
+        },
+    }
+
+    assert FAILED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert dict_to_pretty_json(expected_error_message) in result.output
+
+
+def test_get_batch_status_without_node_url(mocker):
+    """
+    Case: get a batch status by its identifier without passing node URL.
+    Expect: batch status is returned from node on localhost.
+    """
+    batch_id = '6f200995e766da7218ec2a3d0aeabbe1151128063cdf4e954cd08390a879b28e' \
+               '085a06f8708d2e6bb34f6501e8ddc981f0353627c1d4f90c80a656a8090c8751' \
+
+    expected_result = {
+        "result": "COMMITTED",
+    }
+
+    mock_get_batch_status_by_id = mocker.patch('cli.batch.service.loop.run_until_complete')
+    mock_get_batch_status_by_id.return_value = expected_result
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'batch',
+        'get-status',
+        '--id',
+        batch_id,
+    ])
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert expected_result == json.loads(result.output).get('result')
```

Comment:
You mock service, which returns the dictionary. Then this dictionary CLI layer pack to the {result: dict}.

So when you mock, you should mock just a result. Then expect the result in the following dictionary — {result: result}.

So, refactor to:

```
...

committed_status = 'COMMITTED'

mock ..
mock.return_value = committed_status

expected_result = {
    'result': committed_status,
}

assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code assert expected_result == json.loads(result.output)
```

Also, use single quotes.

Reply:
Done.